hbase-handler/src/test/results/positive/hbase_viewjoins.q.out

```diff
@@ -64,9 +64,9 @@ POSTHOOK: type: CREATEVIEW
 POSTHOOK: Input: default@hbase_table_test_1
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@VIEW_HBASE_TABLE_TEST_1
-POSTHOOK: Lineage: VIEW_HBASE_TABLE_TEST_1.ccount SIMPLE [(hbase_table_test_1)hbase_table_test_1.FieldSchema(name:ccount, type:int, comment:), ]
```

Comment:
nit: do you know why these are converted to lowercased, we don't change the HS2 side.

Reply:
This is a Post Executor hook, this hook pulls out the metadata info from HMS and prints it out. So this is a expected change.